apps/package.json

```diff
@@ -119,6 +119,7 @@
     "messageformat": "0.2.2",
     "mkdirp": "0.5.1",
     "mochify": "2.18.0",
+    "mockfirebase": "^0.12.0",
```

Comment:
Since it hasn't hit v1.0 yet, it's more likely that a minor version change to `mockfirebase` may introduce breaking API changes - including whenever https://github.com/katowulf/mockfirebase/issues/108 gets addressed, which would probably break your wrapper.  You may want to pin to 0.12.0 exactly here.

Alternatively, maybe you want tests to break when this gets upgraded, so we'll know they added promise support... but my inclination would be to pin it and manually update, to avoid confusion when it breaks down the road.